contracts/CErc20PluginDelegate.sol

```diff
@@ -36,7 +36,7 @@ contract CErc20PluginDelegate is CErc20Delegate {
 
         require(_plugin != address(0), "0 addr");
 
-        if (address(plugin) != address(0)) {
+        if (address(plugin) != address(0) && plugin.previewRedeem(plugin.balanceOf(address(this))) != 0) {
```

Comment:
Just the balanceOf being !=0 call is enough. Are we over bytecode limit?

Reply:
fixed. over the limit yes